ebs/ebs.go

```diff
@@ -566,6 +566,10 @@ func (d *Driver) BuildVolume(volumeName string, volumeID string, opts map[string
 		return nil, err
 	}
 
+	if mostRecentVolume != nil && *mostRecentVolume.State != ec2.VolumeStateAvailable {
+		return nil, util.NewConvoyDriverErr(fmt.Errorf("Volume=%v is in an unexpected state=%v", volumeID, mostRecentVolume.State), util.ErrVolumeExistsCode)
```

Comment:
Consider adding expected state to the error. Something like:
`fmt.Errorf("Volume=%v is in an unexpected state=%v expected state=%v", volumeID, mostRecentVolume.State, ec2.VolumeStateAvailable)`
This will help the operators in understanding the steps to get the volume to the expected state for recovery.